src/lib/storage/fake_device_factory.rb

```diff
@@ -242,7 +242,7 @@ def create_partition_table(parent, args)
       # @return [Fixnum]
       def str_to_ptable_type(string)
         # Allow different spelling
-        string = "msdos" if string.downcase == "ms-dos"
+        string = "msdos" if string.casecmp("ms-dos").zero?
```

Comment:
when I am thinking about it, you actually would like to do case compare, and downcasing string is just workaround for it. So casecmp do case insensitive comparison, but its value is not so nice. Maybe some helper for String like `case_match?` would be the best.